exammode/forms.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+
+from django.forms import ModelForm, SplitDateTimeField, ModelChoiceField
+
+from .models import ExamSession
+
+from course.models import CourseModule, CourseInstance
+
+
+class ExamSessionForm(ModelForm):
+
+    can_start = SplitDateTimeField(initial=datetime.now())
```

Comment:
Is it really necessary to manually define these fields since they are based on the model? The field type is changed here, which affects the default HTML widget and validation. The HTML widget could be modified in the nested Meta class without defining the field manually, but the SplitDateTimeWidget requires special validation provided by the SplitDateTimeField. So it seems like you have a reason to define the fields manually.

https://docs.djangoproject.com/en/2.2/topics/forms/modelforms/#overriding-the-default-fields
https://docs.djangoproject.com/en/2.2/ref/forms/widgets/#django.forms.SplitDateTimeWidget
https://docs.djangoproject.com/en/2.2/ref/forms/fields/#splitdatetimefield